k8s/namespaces/fees-pay/unprocessed-payment-update/unprocessed-payment-update.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: unprocessed-payment-update
+spec:
+  releaseName: unprocessed-payment-update
+  rollback:
+    enable: true
+    retry: true
+  chart:
+    git: git@github.com:hmcts/hmcts-charts
+    ref: master
+    path: stable/payment-jobs
+  values:
+    job:
+      image: "hmctspublic.azurecr.io/payment/jobs:prod-7f5ce6e-20220704172137" #{"$imagepolicy": "flux-system:unprocessed-payment-update"}
+      schedule: "* * * * *"
```

Comment:
I would change this to a slower schedule, maybe every 30 minutes (*/30 * * * *) as this gets fed through into all the other environments. Keeping the 1minute schedule for testing in demo is fine, but we don't need that in ithc/perftest/aat

Reply:
We will update this schedule once demo testing completes . This schedule is only for testing.